packages/core/test/render3/ivy/jit_spec.ts

```diff
@@ -342,6 +342,32 @@ ivyEnabled && describe('render3 jit', () => {
 
     expect((TestComponent as any).ɵcmp.viewQuery).not.toBeNull();
   });
+
+  describe('invalid parameters', () => {
+    it('should error when creating an @Injectable that extends a class with a faulty parameter', () => {
+      @Injectable({providedIn: 'root'})
+      class Legit {
+      }
+
+
+      @Injectable()
+      class Base {
+        constructor(first: Legit, second: any) {}
+      }
+
+      @Injectable({providedIn: 'root'})
+      class Service extends Base {
+      }
+
+      const ServiceAny = Service as any;
+
+      expect(ServiceAny.ɵprov).toBeDefined();
+      expect(ServiceAny.ɵprov.providedIn).toBe('root');
+      expect(() => ɵɵinject(Service))
+          .toThrowError(
+              /constructor is not compatible with Angular Dependency Injection because its dependency for its parameter 2 is invalid/);
```

Comment:
It seems unexpected to me that we're not using zero-based indices here. Shouldn't it be "parameter at index 1"?

```suggestion
              /constructor is not compatible with Angular Dependency Injection because its dependency  at index 1 is invalid/);
```